src/data/nat/log.lean

```diff
@@ -120,6 +120,17 @@ by { rw ←pow_le_iff_le_log hb (pow_pos (zero_lt_one.trans hb) _),
 lemma log_pos {b n : ℕ} (hb : 1 < b) (hn : b ≤ n) : 0 < log b n :=
 by { rwa [←succ_le_iff, ←pow_le_iff_le_log hb (hb.le.trans hn), pow_one] }
 
+lemma log_mul (b n : ℕ) (hb : 1 < b) (hn : 0 < n) : log b (n * b) = log b n + 1 :=
```

Comment:
```suggestion
lemma log_mul_self (b n : ℕ) (hb : 1 < b) (hn : 0 < n) : log b (n * b) = log b n + 1 :=
```
I would expect `log_mul` to be about `log b (m * n)`.

Reply:
I used `log_mul_base` to be more clear about what `self` is being discussed.